.idea/misc.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="Black">
+    <option name="enabledOnReformat" value="true" />
+    <option name="enabledOnSave" value="true" />
+    <option name="pathToExecutable" value="/opt/homebrew/bin/black" />
```

Comment:
This shouldn't be pushed, not everyone uses homebrew 🙂

Reply:
Well I pushed it on purpose, without it my black formatting in the IDE doesn't work. So either this has to be here in some form, or this entire file shouldn't be here. Unless you have some other fix for me?